admin/spec/components/previews/solidus_admin/ui/badge/component_preview.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# @component "ui/badge"
+class SolidusAdmin::UI::Badge::ComponentPreview < ViewComponent::Preview
+  layout "solidus_admin/preview"
+  include SolidusAdmin::ContainerHelper
```

Comment:
What do you think about moving it into a `SolidusAdmin::BaseComponentPreview` class? I guess we'll need those for all other previews...

Reply:
This bothers me as well, 😞 and I tried, but for some reason the preview broke, after a while I stopped spending time on it and just added the two lines to the generator. I think over time we might give it another crack and see if we succeed. 

I can try again if you think this is needed right away.

Reply:
It's not needed now, but it probably will be needed for the next one :slightly_smiling_face: We can work on it separately, though.